Type.sbvr

```diff
@@ -8,6 +8,9 @@ Term:       Time
 Term:       Interval
 Term:       File
 
+Term:       Integer
```

Comment:
Integer is defined above as well, isn’t it an error to redefine it? Also, making integer a big int, won’t it require a migration in the api?

Reply:
Urg, I got this the wrong way round, fixed